src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPages(from: number, to: number): number[] {
+  const numbers = [];
+
+  for (let n = from; n <= to; n += 1) {
```

Comment:
Avoid using a 'for' loop to create a new array with modified values. Instead, use 'map'.